app/frontend/src/features/communities/CommunityPage/CommunityPage.tsx

```diff
@@ -19,33 +19,16 @@ export const useCommunityPageStyles = makeStyles((theme) => ({
     marginBottom: theme.spacing(1),
   },
   cardContainer: {
-    alignItems: "flex-start",
-    [theme.breakpoints.down("xs")]: {
-      //break out of page padding
-      left: "50%",
-      marginLeft: "-50vw",
-      marginRight: "-50vw",
-      position: "relative",
-      right: "50%",
-      width: "100vw",
-    },
```

Comment:
Not a fan of this? It allowed the horizontal scroller to go all the way to the edge

Reply:
ooops, missed that. Will do another commit later this night.